source/draw/RasterYuv420Semiplanar.ooc

```diff
@@ -29,19 +29,9 @@ RasterYuv420Semiplanar: class extends RasterImage {
 	y ::= this _y
 	uv ::= this _uv
 	stride ::= this _y stride
-	init: func ~fromRasterImages (yImage: RasterMonochrome, uvImage: RasterUv) {
-		super(yImage size)
-		this _y = yImage
-		this _y referenceCount increase()
-		this _uv = uvImage
-		this _uv referenceCount increase()
-	}
-	init: func ~fromYuvSemiplanar (original: This, y: RasterMonochrome, uv: RasterUv) {
-		super(original)
-		this _y = y
-		this _y referenceCount increase()
-		this _uv = uv
-		this _uv referenceCount increase()
+	init: func ~fromRasterImages (=_y, =_uv) {
+		super(this _y size)
+		(this _y, this _uv) referenceCount increase()
 	}
```

Comment:
This merge assumes that no one uses the stride from the YUV image as a whole since the getter for stride in RasterImage is meant to be overridden in RasterPacked. Removing stride from Image would however break the API.